server/routes/charts.js

```diff
@@ -213,4 +213,30 @@ router
       return res.status(404).json({ message: 'Chart was not duplicated' })
     }
   })
+
+router
+  .route('/api/v1/charts/:chart_id/shared')
+  .patch(ensureAuthenticated, async (req, res) => {
+    try {
+      const { chart_id } = req.params
+      const { dataDb } = req.app.locals
+      const { sharedWith } = req.body
+      const { result } = await dataDb.collection(collections.charts).updateOne(
+        { _id: new ObjectID(chart_id) },
+        {
+          $set: {
+            sharedWith,
```

Comment:
does this mean that a chart can only be shared with 1 user? Is that how configurations work?

Reply:
Configurations has an array of users. For Charts we also have an array of users, this is what it looks like on the db.   
 "owner" : "blockedUser",
    "sharedWith" : [
        "dpdash"
    ],